silk/tools/otns_manager.py

```diff
@@ -108,10 +108,20 @@ def set_title(self, title: str, x=0, y=20, font_size=20):
       y (int): y position of title.
       font_size (int): font size of title.
     """
-    response = self.stub.CtrlSetTitle(
-        visualize_grpc_pb2.SetTitleEvent(
-            title=title, x=x, y=y, font_size=font_size))
-    self.logger.info("Sent title {:s}, response: {}".format(title, response))
+    response = self.stub.Command(
+        visualize_grpc_pb2.CommandRequest(
+            command=f"title \"{title}\" x {x} y {y} fs {font_size}"))
+    self.logger.info(f"Sent title {title}, resp: {response}".rstrip("\n"))
+
+  def set_speed(self, speed: float):
+    """Send test replay speed to OTNS.
+
+    Args:
+        speed (float): test replay speed.
+    """
+    response = self.stub.Command(
+        visualize_grpc_pb2.CommandRequest(command=f"speed {speed}"))
+    self.logger.info(f"Sent speed {speed}, resp: {response}".rstrip("\n"))
```

Comment:
```suggestion
    self.run_command(f"speed {speed}")
```

Have one single method for running command since it's going to be used very often.